src/services/genre_detector.py

```
@@ -2,36 +2,53 @@
             for genre, keywords in self.genre_keywords.items():
                 genre_scores[genre] = sum(1 for keyword in keywords if keyword in text)
-            
+
             max_score = max(genre_scores.values())
             if max_score > 0:
                 return max(genre_scores.items(), key=lambda x: x[1])[0]
-            return 'Other'
-            
+            return "Other"
+
         except Exception as e:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)